.travis.yml

```diff
@@ -62,24 +62,39 @@ matrix:
         - FEDORA_CI=Y
       services:
         - docker
+    - jdk: oraclejdk9
+      addons:
+        apt:
+          packages:
+            - oracle-java9-installer # Test with latest update
+        postgresql: "9.6"
+      env:
+        - PG_VERSION=9.6
+        - JDK=9
     - jdk: oraclejdk8
       sudo: required
-      dist: trusty
       env:
         - PG_VERSION=HEAD
         - XA=true
         - REPLICATION=Y
     - jdk: oraclejdk8
       sudo: required
-      dist: trusty
+      before_install:
+        - sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9
+        - sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main'
+        - sudo apt-get update -qq && sudo apt-get install zulu-8 -y
+        - export JDK8_HOME=/usr/lib/jvm/zulu-8-amd64
```

Comment:
Travis results would read "oraclejdk8", however it would use Zulu8. Would you add an environment variable to tell the difference?